src/main/java/org/deepsymmetry/beatlink/VirtualRekordbox.java

```diff
@@ -0,0 +1,2711 @@
+package org.deepsymmetry.beatlink;
```

Comment:
This class started as a copy and paste of VirtualCdj. I am working now to clean things up to adjust comments and remove anything that is not applicable to this class.

Reply:
Ok, we can go that way but we want to be thorough about removing things, it is probably going to end up a fairly small and focused class.